lib/src/seq.rs

```diff
@@ -39,6 +39,20 @@ macro_rules! declare_seq_with_contents_constraints_impl {
                 }
             }
 
+            #[cfg_attr(feature="use_attributes", unsafe_hacspec)]
+            pub fn with_capacity(l: usize) -> Self {
+                Self {
+                    b: Vec::with_capacity(l),
+                }
+            }
+
+            #[cfg_attr(feature="use_attributes", unsafe_hacspec)]
+            #[inline(always)]
+            pub fn reserve(mut self, additional: usize) -> Self {
```

Comment:
I can see why you would need that for performance improvements, but there should be a comment saying that this is equivalent to the identity function semantics-wise because the formal model of hacspec does not know that its `Seq`s are resizable arrays.